plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -176,18 +174,18 @@ public void testColumnsCached()
         JdbcTableHandle table = getAnyTable(schema);
         JdbcColumnHandle phantomColumn = addColumn(table);
 
-        int expectedLoad = 0, expectedHit = 0, expectedMiss = 0;
+        AssertedStats stats = new AssertedStats();
 
         // Read column into cache
         assertThat(cachingJdbcClient.getColumns(SESSION, table)).contains(phantomColumn);
-        assertCacheLoadsHitsAndMisses(cachingJdbcClient.getColumnsCacheStats(), expectedLoad += 1, expectedHit, expectedMiss += 1);
+        assertCacheLoadsHitsAndMisses(cachingJdbcClient.getColumnsCacheStats(), stats.addLoads(1).addMisses(1));
```

Comment:
Why do we want both of this methods: `addLoads()` and `addMisses()`?

Reply:
What do you propose instead? They are increasing different counters. This change does not decrease code readability

Reply:
Just asking